extensions/git-commands/src/search-commands.tsx

```diff
@@ -94,7 +103,7 @@ export default function Command() {
         title={name}
         subtitle={{ value: command, tooltip: command }}
         detail={<List.Item.Detail markdown={detail} />}
-        keywords={[description, command]}
+        keywords={[command, ...commandKeywords(command)]}
```

Comment:
Question (non-blocking): why not keep description in filtering keywords as well? Will it result in too much noise?

Reply:
@jfkisafk Hi, I noticed some details regarding the search in the description.

### Example:

```
git
The base command for all git commands.
```
If you search with the word `base`, you should find `git` but this does not happen. You must type `the base` to find it.

I guess I should convert the description to an **Array with the words**. And I thought about the **performance** or the correct way to do it. It must be something very common in Raycast, I don't want to invent anything.

It would really be much better if you can search by description. I'm going to document myself and make it happen.

Reply:
ok, I just added it :)

Reply:
cool cool I think the searchBarPlaceholder still makes sense:

https://github.com/raycast/extensions/blob/0ecee03c8af090a693061e8642073a7ebcd91fe0/extensions/git-commands/src/search-commands.tsx#L166

Feel free to resolve this comment.